graylog2-rest-client/src/main/java/org/graylog2/restclient/lib/ApiClientImpl.java

```diff
@@ -725,5 +730,83 @@ public URL prepareUrl(ClusterEntity target) {
                 throw new RuntimeException(e);
             }
         }
+
+        @Override
+        public InputStream executeStreaming() throws APIException, IOException {
+            if (radio != null && (node != null || nodes != null)) {
+                throw new RuntimeException("You set both and a Node and a Radio as target. This is not possible.");
+            }
+
+            final ClusterEntity target;
+
+            if (radio == null) {
+                if (node == null) {
+                    if (nodes != null) {
+                        LOG.error("Multiple nodes are set, but execute() was called. This is most likely a bug and you meant to call executeOnAll()!", new Throwable());
```

Comment:
So you can see which call site made the programming error. This is a help for the developer.